lib/cli.js

```diff
@@ -184,8 +184,8 @@ CLI.prototype.scanCommands = function scanCommands(dir) {
 			var file = isDir ? path.join(dir, filename) : filename;
 			// we don't allow commands that start with _ or have spaces
 			if (fs.existsSync(file) && fs.statSync(file).isFile() && jsfile.test(filename) && (!isDir || !ignore.test(path.basename(file)))) {
-				// we don't allow commands that start with _ or have spaces
-				var name = filename.replace(jsfile, '').toLowerCase();
+				var command = require(file);
```

Comment:
Before blindly require()'ing the command file, perhaps we should check that it's syntactically sane like we do in the context.js: https://github.com/appcelerator/titanium/blob/master/lib/context.js#L311-L320.